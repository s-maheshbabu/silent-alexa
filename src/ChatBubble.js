import React from "react";
import { ChatBubble } from "react-chat-ui";

export default function(props) {
  return (
    <ChatBubble
      message={props.message}
      bubbleStyles={{
        text: {
          fontSize: 20
        },
        chatbubble: {
          background: "mediumslateblue"
        },
        userBubble: {
          background: "lightcoral"
        }
      }}
    />
  );
}
