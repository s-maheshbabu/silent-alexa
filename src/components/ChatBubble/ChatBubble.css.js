export default {
  bubbleStyles: {
    text: {
      fontSize: 20
    },
    chatbubble: {
      background: "#00ACE0"
    },
    userBubble: {
      background: "#646A72"
    }
  }
};
