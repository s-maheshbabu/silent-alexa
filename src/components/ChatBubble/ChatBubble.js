import React from "react";
import { ChatBubble } from "monkas-chat";

import styles from "./ChatBubble.css.js";

export default function(props) {
  return (
    <ChatBubble
      message={props.message}
      bubbleStyles={{
        ...styles.bubbleStyles
      }}
    />
  );
}
