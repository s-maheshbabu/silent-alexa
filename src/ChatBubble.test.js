import React from "react";
import { shallow } from "enzyme";
import ChatBubble from "ChatBubble";

it("renders correctly (snapshot testing)", () => {
  const wrapper = shallow(<ChatBubble />);
  expect(wrapper).toMatchSnapshot();

  wrapper.unmount();
});
