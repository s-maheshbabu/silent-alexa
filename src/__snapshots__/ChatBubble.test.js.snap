// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly (snapshot testing) 1`] = `
<ChatBubble
  bubbleStyles={
    Object {
      "chatbubble": Object {
        "background": "mediumslateblue",
      },
      "text": Object {
        "fontSize": 20,
      },
      "userBubble": Object {
        "background": "lightcoral",
      },
    }
  }
/>
`;
